networkx/algorithms/shortest_paths/weighted.py

```diff
@@ -584,6 +584,23 @@ def bellman_ford(G, source, weight='weight'):
     if len(G) == 1:
         return pred, dist
 
+    return _bellman_ford_relaxation(G, pred, dist, [source], weight)
+
+
+def _bellman_ford_relaxation(G, pred, dist, source_list, weight):
+    """
+    Relaxation loop for Bellman - Ford algorithm.
+
+    :param G: NetworkX graph
+    :param pred: dictionary keyed by node to predecessor in the
+    path
+    :param dist: dictionary keyed by node to distance from the source
+    :param source_list: list of nodes setting as source nodes
+    :param weight: Edge data key corresponding to the edge weight
+    :return: dictionaries
+       Returns two dictionaries keyed by node to predecessor in the
+       path and to the distance from the source respectively.
```

Comment:
This function is supposed to be auxiliary. This is the reason why I did not  follow the official way of formatting the docstring.